src/esw/brushed_motors/ROSHandler.cpp

```diff
@@ -106,3 +181,126 @@ void ROSHandler::moveMastGimbal(const mrover::MastGimbal::ConstPtr& msg) {
     moveControllerOpenLoop("mast_gimbal_up_down", (float) msg->up_down);
     moveControllerOpenLoop("mast_gimbal_left_right", (float) msg->left_right);
 }
+
+// REQUIRES: valid req and res objects
+// MODIFIES: res
+// EFFECTS: sends a move/calibration command to the mcu
+bool ROSHandler::processMotorCalibrate(mrover::CalibrateMotors::Request& req, mrover::CalibrateMotors::Response& res) {
+    auto controller_iter = ControllerMap::controllersByName.find(req.name);
+
+    if (controller_iter == ControllerMap::controllersByName.end()) {
+        ROS_ERROR("Could not find controller named %s.", req.name.c_str());
+        res.actively_calibrating = false;
+        return false;
```

Comment:
I think we want to return true always. See http://wiki.ros.org/roscpp/Overview/Services section 3.2

If we return false then nothing will be sent back to teleop